### PR TITLE
Prevent 'null' from displaying in the 'Found similar people' table.

### PR DESCRIPTION
**Pull request by wikumChamith**

### CODE REVIEW

File: omod/src/main/webapp/resources/scripts/dojo/src/widget/openmrs/OpenmrsSearch.js [hunk 1]
```diff
@@ -475,9 +475,7 @@ dojo.widget.defineWidget(
 
 
 	noCell: function() {
-		var td = document.createElement("td");
```
Comment (dkayiwa) [inline omod/src/main/webapp/resources/scripts/dojo/src/widget/openmrs/OpenmrsSearch.js:478]: Did you intentionally remove this?

Comment (wikumChamith) [reply omod/src/main/webapp/resources/scripts/dojo/src/widget/openmrs/OpenmrsSearch.js:478]: Yes. By setting the display value of the element to "none" we get a table similar to this.

Screenshot from 2024-10-02 19-49-16

Comment (dkayiwa) [reply omod/src/main/webapp/resources/scripts/dojo/src/widget/openmrs/OpenmrsSearch.js:478]: You mean it has been like that and no body has ever seen it? Or was it caused by something else?

Comment (wikumChamith) [reply omod/src/main/webapp/resources/scripts/dojo/src/widget/openmrs/OpenmrsSearch.js:478]: I think so. That's how HTML works.

```
    <table border="1">
        <tbody>
        <tr>
            <td>Hello</td>
            <td>World</td>
            <td>Hello</td>
        </tr>
        <tr>
            <td>Hello</td>
            <td style="display: none"></td>
            <td>Hello</td>
        </tr>
        </tbody>
    </table>
```

For example this HTML code outputs this:

Screenshot from 2024-10-02 20-23-08

**Pull request merged**